src/chainparamsbase.cpp

```diff
@@ -73,11 +35,11 @@ const CBaseChainParams& BaseParams()
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain)
 {
     if (chain == CBaseChainParams::MAIN)
-        return std::unique_ptr<CBaseChainParams>(new CBaseMainParams());
+        return std::unique_ptr<CBaseChainParams>(new CBaseChainParams("", 8332));
```

Comment:
nit: Could use `MakeUnique<CBaseChainParams>("", 8332)`

Reply:
I tried, but it seems it's a C++14 feature, not C++11. :(

Reply:
c++14 provides make_unique but MakeUnique is just a function in util:

https://github.com/bitcoin/bitcoin/blob/0910cbe4ef31eb95fd76c7c2f820419fe64a3150/src/util.h#L336